### DISCUSSION TITLE
Issue with running setup.

### DISCUSSION THREAD

**Discussion by Jasepi** [cyberofficial/Synthalingua#39] [Q&A]
Ive followed as far as i know correctly all the setup, everything is installed, python path is setup correctly and ive restarted system as well. 
whenever i try to run the setup i get this error. ive triple checked and im pretty sure its set up correctly i cannot find whats causing this problem.
![image](https://github.com/cyberofficial/Synthalingua/assets/120144478/17e31abf-71b6-47f7-8724-9178ff1ecd28)

**Comment by cyberofficial** [comment]
In the `requirements_static.txt` file, can you edit the line `openai-whisper @ git+https://github.com/openai/whisper.git@b5851c6c40e753606765ac45b85b298e3ae9e00d` and replace with  `git+https://github.com/openai/whisper.git` and rerun the set up?

**Reply by Jasepi** [reply]
that did work, it got further, but it is now explicitly listing that it cannot run on python 3.11.4 so my python version may just be wrong then. 
![image](https://github.com/cyberofficial/Synthalingua/assets/120144478/c704b3f8-1ac1-439d-95d7-54d68935e816)

**Comment by cyberofficial** [comment]
As per installation, requires python 3.10.9+ but less than 3.11

[Installation](https://github.com/cyberofficial/Synthalingua#installation)
Download and install [Python 3.10.9](https://www.python.org/downloads/release/python-3109/).
Make sure to check the box that says "Add Python to PATH" when installing. If you don't check the box, you will have to manually add Python to your PATH. You can check this guide: [How to add Python to PATH](https://datatofish.com/add-python-to-windows-path/).
You can choose any python version that is 3.10.9 up to the latest version. The tool will -not- work on any python version that is 3.11 or higher. Must be 3.10.9+ not 3.11.x.

**Comment by cyberofficial** [comment]
Hello are you still having issues with this?
